nuxt/components/formular.vue

```diff
@@ -228,11 +228,12 @@ export default {
     handleSubmit(e) {
       if (!this.validate()) {
         e.preventDefault();
+        this.$emit('success');
```

Comment:
Isn't this the error case? So when validate is false. Should throw an error shouldn't it?